charts/catalog/README.md

```diff
@@ -0,0 +1,68 @@
+# Service Catalog
+
+Service Catalog is a Kubernetes Incubator project that provides a
+Kubernetes-native workflow for integrating with [Open Service Brokers]
+(https://www.openservicebrokerapi.org/) to provision and bind to application
+dependencies like databases, object storage, message-oriented middleware, and
+more.
+
+For more information, [visit the project on github]
+(https://github.com/kubernetes-incubator/service-catalog).
+
+## Prerequisites
+
+- Kubernetes 1.5+ with Beta APIs enabled
```

Comment:
Is this still true? I saw something about 1.6 beta / alpha floating around for being able to do something. (on the slack channel).